resources/assets/js/src/components/chart/Chart.tsx

```diff
@@ -86,7 +89,7 @@ export default class Chart extends React.Component<IProps, IState> {
                         trackMouse={true}
                     >
                         <ResponsiveContainer width="100%" height={300}>
-                            <LineChart width={830} height={400} data={data}>
+                            <LineChart width={830} height={400} data={this.mapDatesToShowingFormat(this.state.data)}>
```

Comment:
Those are initial width and height?

Reply:
yes, it is required by library, even it's handled by responsive container (but without these variables it doesn't initialize)